elements/lisk-bft/src/utils/validators.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { Validator, CONSENSUS_STATE_VALIDATORS_KEY, validatorsSchema } from '@liskhq/lisk-chain';
+import { codec } from '@liskhq/lisk-codec';
+import { StateStore } from '../types';
+
+export const getValidators = async (stateStore: StateStore): Promise<Validator[]> => {
+	const validatorsBuffer = await stateStore.consensus.get(CONSENSUS_STATE_VALIDATORS_KEY);
+		if (!validatorsBuffer) {
+			throw new Error('Validator set must exist');
+		}
+		const { validators } = codec.decode<{ validators: Validator[]}>(validatorsSchema, validatorsBuffer);
+
+		return validators;
+};
```

Comment:
Can we move this as pure function to the `lisk-chain` and export from there? Then we don't need to deal with this complexity only on once place.